incident-process.md

```diff
@@ -96,5 +96,5 @@ Incidents and their results are confidential to RLS Administrators and those inv
 
 ## Administrators
 
-The current roster of Administrators is maintained in the primary [Code of Conduct](https://github.com/randsleadershipslack/documents-and-resources/blob/master/code-of-conduct.md#administrators) document.
+The current roster of Administrators is maintained in the primary [Code of Conduct](https://github.com/randsleadershipslack/documents-and-resources/blob/master/code-of-conduct.md#administrators) document. 
```

Comment:
I can cut this space - was considering adding something about the bookmark/pins in #rls-admins as well, which also note the list of admins, for convenience.